src/const.ts

```diff
@@ -16,10 +17,7 @@ export const TEN = new BN(10)
 export const ALLOWANCE_MAX_VALUE = TWO.pow(new BN(256)).sub(ONE) // 115792089237316195423570985008687907853269984665640564039457584007913129639935
 // Arbitrarily big number for checking if the token is enabled
 export const ALLOWANCE_FOR_ENABLED_TOKEN = TWO.pow(new BN(128)) // 340282366920938463463374607431768211456
```

Comment:
should this also be subtracted by one? I don't think you can approve that high a value